### PR TITLE
Include gradelw.bat and update to grade 4.8

### PR DESCRIPTION
**Pull request by kerkhofsd**
- Include gradlew.bat to build on Windows
- Update to gradle 4.8

## Related Issue
No related issues, just maintenance commits.

## Motivation and Context
gradlew.bat to be able to build on a Windows machine. Gradle to 4.8 to keep up to date.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,6 @@
 .settings
 .project
 *.sh
-*.bat
```
Comment (yregaieg) [inline .gitignore]: You could force version the bat file, even without loosing this line

**Comment by kerkhofsd** [comment]
@yregaieg updated. Thanks for the feedback.

**Comment by kerkhofsd** [comment]
@todorinskiz can you test building with the gradlew.bat and merge?

File: .gitignore (hunk 1 above)
Comment (todorinskiz) [reply .gitignore]: Like so: `git add --force file.foo`
But why would you introduce such ambiguity? One will see .bat file in the repo and just commit without reviewing, then their .bat is gone on the next checkout.  If there are .bat files to be excluded from the PR, its part of the review to ask the author to remove them (or do it yourself)?

**Pull request merged**